charts/civil-sdt/values.preview.template.yaml

```diff
@@ -12,6 +12,17 @@ java:
   postgresql:
     enabled: true
   keyVaults:
+    civil:
+      resourceGroup: civil
+      secrets:
+        - name: civil-sdt-service-s2s-secret
+          alias: CIVIL_SDT_S2S_KEY
+        - name: civil-sdt-POSTGRES-USER
+          alias: SDT_DB_USERNAME
+        - name: civil-sdt-POSTGRES-PASS
+          alias: SDT_DB_PASSWORD
+        - name: civil-sdt-servicebus-connection-string
+          alias: CIVIL_SDT_SERVICEBUS_CONNECTION_STRING
     civil-sdt:
       disabled: true
```

Comment:
Should only be one key vault which is the one named civil-sdt.  The resourceGroup parameter isn't needed.

Note that enabling the key vault in preview means that the build will fail if a new secret is being created by the pull request.  This is because the plan isn't applied in preview build pipeline, so the secret won't exist.

Reply:
removed resourceGroup: civil